packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -318,6 +319,56 @@ describe("SharedString", () => {
             sharedString2.connect(services2);
         });
 
+        it("can maintain interval consistency", () => {
+            const collection1 = sharedString.getIntervalCollection("test");
+            sharedString.insertText(0, "xyz");
+            containerRuntimeFactory.processAllMessages();
+            const collection2 = sharedString2.getIntervalCollection("test");
+            assert.notStrictEqual(collection2, undefined, "undefined");
+            assert.strictEqual(sharedString.getText(), sharedString2.getText(), "not equal text");
+
+            sharedString.insertText(0, "abc");
+            let interval = collection1.add(1, 1, IntervalType.SlideOnRemove);
+            const intervalId = interval.getIntervalId();
+            sharedString2.insertText(0, "wha");
+
+            containerRuntimeFactory.processAllMessages();
+            assert.strictEqual(sharedString.getText(), "whaabcxyz", "different text 1");
+            assert.strictEqual(sharedString.getText(), "whaabcxyz", "different text 2");
+
+            let testInterval = collection1.getIntervalById(intervalId);
+            for (interval of collection1) {
```

Comment:
If there's a bug and the collections are empty, this test will erroneously pass.

Reply:
Good point.